parsers/test/test_ERCOT.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+"""Tests for US_MISO.py"""
+
+import json
+import logging
+import os
+import unittest
+from datetime import datetime
+from unittest.mock import patch
+
+import pytz
+from arrow import get
+from testfixtures import LogCapture
+
+from parsers import US_ERCOT
+
+
+class TestUSERCOT(unittest.TestCase):
+    """Patches in a fake response from the data source to allow repeatable testing."""
+
+    def test_fetch_production(self):
```

Comment:
Since you are using the TestCase class, I would recommend using the adaptor and session for passing patches, it's a bit cleaner.
Have a look here for inspiration:
https://github.com/electricitymaps/electricitymaps-contrib/blob/2005e686ed2a27a1e6af815f20afb0abd6981858/parsers/test/test_NTESMO.py#L14